### PR TITLE
[sitecore-jss-proxy][sitecore-jss-nextjs] Fix for getCSPHeader function

### PR DESCRIPTION
**Pull request by yavorsk**
- [x] This PR follows the [Contribution Guide](https://github.com/Sitecore/jss/blob/dev/CONTRIBUTING.md)
- [x] Changelog updated
- [ ] Upgrade guide entry added

## Description / Motivation

This PR fixes the getCSPHeader function so that it returns proper value for the CSP header when JSS_ALLOWED_ORIGINS lists multiple origins delimited with comma.

## Testing Details

- [x] Unit Test Added
- [ ] Manual Test/Other (Please elaborate)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by stasmaxymov** [comment]
@yavorsk It seem analogues function from NextJs package should have the same problem:
https://github.com/Sitecore/jss/blob/0c8193ab38c6cd4e7aafb151741b1401f8427b71/packages/sitecore-jss-nextjs/src/editing/editing-render-middleware.ts#L407

File: packages/sitecore-jss-proxy/src/middleware/editing/render.test.ts [hunk 1]
```diff
@@ -459,3 +460,28 @@ describe('editingRouter - /editing/render', () => {
...
+  it('should return tha value of the CSP header', () => {
+    process.env.JSS_ALLOWED_ORIGINS = 'https://pages-dev.sitecore-staging.cloud';
+    const expectedHeader = `frame-ancestors 'self' https://pages-dev.sitecore-staging.cloud ${EDITING_ALLOWED_ORIGINS.join(
+      ' '
+    )}`;
+    const actualHeader = getSCPHeader();
+    expect(actualHeader).to.equal(expectedHeader);
+  });
+
+  it('should return tha value of the CSP header when multiple origins are listed in JSS_ALLOWED_ORIGINS', () => {
```
Comment (illiakovalenko) [inline packages/sitecore-jss-proxy/src/middleware/editing/render.test.ts]:
```suggestion
  it('should return the value of the CSP header when multiple origins are listed in JSS_ALLOWED_ORIGINS', () => {
```

File: packages/sitecore-jss-proxy/src/middleware/editing/render.test.ts [hunk 2]
```diff
@@ -459,3 +460,28 @@ describe('editingRouter - /editing/render', () => {
       .end(done);
   });
 });
+
+describe('getSCPHeader', () => {
+  afterEach(() => {
+    delete process.env.JSS_ALLOWED_ORIGINS;
+  });
+
+  it('should return tha value of the CSP header', () => {
```
Comment (illiakovalenko) [inline packages/sitecore-jss-proxy/src/middleware/editing/render.test.ts]:
```suggestion
  it('should return the value of the CSP header', () => {
```

**Review by illiakovalenko — APPROVED** [review]
👍

**Pull request merged**